packages/web-components/@sfx/search-box/src/search-box.ts

```diff
@@ -172,14 +172,18 @@ export default class SearchBox extends Base {
     return html`
       <style>
         sfx-search-box {
-          display: inline-block;
+          display: flex;
```

Comment:
This should preserve the `inline` aspect.

```suggestion
          display: inline-flex;
```